taxonomy-editor-frontend/src/pages/editentry/ListAllProperties.jsx

```diff
@@ -1,19 +1,30 @@
 import { IconButton, TextField, Typography } from "@mui/material";
 import AddBoxIcon from '@mui/icons-material/AddBox';
-import { DataGrid } from '@mui/x-data-grid';
+import MaterialTable from '@material-table/core';
+import { useState } from "react";
 
 const ListAllProperties = ({ nodeObject, setNodeObject }) => {
     let toBeRendered = []
+
+    function guidGenerator() {
```

Comment:
don't understand what's the value of doing this?

Reply:
It's to generate a unique key for each row of the properties table.

Reply:
thanks, I see, mmm,  if you need this, possibly the place to do it is when loading the data or inside a `useEffect` as proposed below, that way you know when it will run and why it will run. Also you would need to do it on every action (delete, add new item)...
(think [this](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) may help)